<a href="https://colab.research.google.com/github/syuan611/115-1/blob/main/411403930_H1.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [1]:
x=50000*(1+0.015)**5
print(x)

53864.20019421873


題目：銀行存款複利的計算，假設目前銀行年利率是1.5%，複利公式如下：

本金和= 本金*(1+年利率)n  # n是年

假設有一筆5萬元，請計算5年後的本金和？

輸出範例：
53864.20019421873